src/Http/Http/src/Timeouts/IHttpRequestTimeoutFeature.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Timeouts;
+
+/// <summary>
+/// 
+/// </summary>
+public interface IHttpRequestTimeoutFeature
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    CancellationToken RequestTimeoutToken { get; }
```

Comment:
Do we also want to discuss the plan for reducing the allocation for the CTS? @davidfowl and @JamesNK had some ideas on how to do it. I'm just wondering that with that implementing we do have this cancellation token to expose or not.

Reply:
We can follow up on that separately, this PR is functionally complete and performing well on the benchmarks.